streaming/src/main/scala/org/apache/gearpump/streaming/appmaster/ExecutorManager.scala

```diff
@@ -44,11 +44,11 @@ import scala.util.Try
  * @param appContext
  * @param executorFactory
  */
-private[appmaster] class ExecutorManager (
-    userConfig: UserConfig,
-    appContext: AppMasterContext,
-    executorFactory: (ExecutorContext, UserConfig, Address, ExecutorId) => Props,
-    clusterConfig: Config)
+private[appmaster] class ExecutorManager(userConfig: UserConfig,
+                                         appContext: AppMasterContext,
```

Comment:
About the style, check http://docs.scala-lang.org/style/declarations.html#classes